pkg/core/config/tenant.go

```diff
@@ -420,6 +420,11 @@ func (c *TenantConfiguration) AfterUnmarshal() {
 		c.AppConfig.UserVerification.Criteria = UserVerificationCriteriaAny
 	}
 	for i, config := range c.AppConfig.UserVerification.LoginIDKeys {
+		// UpdateNilFieldsWithZeroValue by design only handle ptr and map attributes
```

Comment:
Would it be better to also support slices in `UpdateNilFieldsWithZeroValue`? Handling pointers and maps only seems arbitrary.

Reply:
Updated!